net.tabletopclub.TabletopClub.json

```diff
@@ -0,0 +1,59 @@
+{
+    "app-id": "net.tabletopclub.TabletopClub",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "tabletop-club-wrapper",
+    "finish-args": [
+        "--socket=x11",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--filesystem=xdg-documents/TabletopClub:create"
+    ],
+    "modules": [
+        {
+            "name": "tabletop-club",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D -m 755 -p TabletopClub.x86_64 /app/bin/tabletop-club",
+                "install -D -m 755 -p tabletop-club-wrapper.sh /app/bin/tabletop-club-wrapper",
+                "install -D -m 755 -p -t /app/lib/ libwebrtc_native.linux.release.64.so",
```

Comment:
why is this library being made executable?

Reply:
Apologies, I thought that libraries needed to be marked with an executable bit for them to load.